src/base/server_type_enum.hpp

```diff
@@ -33,4 +33,34 @@ enum class server_type_config
   PAGE,
   SINGLE_NODE,
 };
+
+inline const char *
+server_type_to_string (SERVER_TYPE type)
+{
+  switch (type)
+    {
+    case SERVER_TYPE_UNKNOWN:
+      return  "unknown";
+    case SERVER_TYPE_TRANSACTION:
+      return "transaction";
+    case SERVER_TYPE_PAGE:
+      return "page";
+    case SERVER_TYPE_ANY:
+      return "any";
```

Comment:
```suggestion
      return "any";
    default:
      assert (false);
```
to guard that future modifications of the enum do not leave this code behind (without `default` would still be signalled by a compiler warning but those do test to be overlooked)